instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/internal/OpenTelemetryStatement.java

```diff
@@ -43,7 +45,8 @@ public class OpenTelemetryStatement<S extends Statement> implements Statement {
   protected final String query;
   protected final Instrumenter<DbRequest, Void> instrumenter;
 
-  private final ArrayList<String> batchCommands = new ArrayList<>();
+  private final Collection<String> batchCommands = new LinkedHashSet<>();
```

Comment:
I didn't follow why a Set?

Reply:
I went with a set because user could add the same statement multiple times. I did the javaagent part first where this was not implemented and changed here to match the agent instrumentation. Could change to list as having the same statement in the batch multiple times should be very rare, I guess for that to make sense the statement would need to use `rand()` or something similar. These non-unique statements would get collapsed after the sanitization anyway.

Reply:
thanks, if there's not much difference, I'd suggest List just to make the code more obvious

Reply:
replaced `Set` with `List`